### PR TITLE
feat: fix restore banner theme issues

### PR DESCRIPTION
**Pull request by AmanNegi**
# Issue
- Change button text color.
- Add Hover feature improving the UX.

> Fixes: #3276 

#### Earlier
https://github.com/AppFlowy-IO/AppFlowy/assets/37607224/abef4549-dd6a-4e3e-b586-e5ed9bade743

#### Now
light-dark-banner

#### PR Checklist

- [x] My code adheres to the [AppFlowy Style Guide](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/software-contributions/submitting-code/style-guides)
- [x] I've listed at least one issue that this PR fixes in the description above.
- [x] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [x] All existing tests are passing.

### CODE REVIEW

File: frontend/appflowy_flutter/lib/plugins/document/presentation/banner.dart [hunk 1]
```diff
@@ -54,15 +55,15 @@ class DocumentBanner extends StatelessWidget {
...
+                hoverColor: Colors.white.withOpacity(0.3),
                 highlightColor: Theme.of(context).colorScheme.primary,
                 outlineColor: Colors.white,
                 borderRadius: Corners.s8Border,
                 onPressed: onDelete,
                 child: FlowyText.medium(
                   LocaleKeys.deletePagePrompt_deletePermanent.tr(),
-                  color: Theme.of(context).colorScheme.onPrimary,
-                  fontSize: 14,
+                  color: Colors.white,
```
[the remark this answers is not in the document]
Comment (AmanNegi) [reply frontend/appflowy_flutter/lib/plugins/document/presentation/banner.dart:65]: I also thought so, but I have seen it being used quite often so I thought it was fine. I'll look for what property of Theme would serve our purpose here.

**Comment by AmanNegi** [comment]
I have now simply updated the logic to:
- Light Mode: Black Color
- Dark Mode: White Color

## Updated Preview:

https://github.com/AppFlowy-IO/AppFlowy/assets/37607224/a9634e52-8ad9-4767-a857-fb089b000da7

**Comment by AmanNegi** [comment]
Thanks for the help @hyj1204. I have updated the code as suggested. 

## Updated Preview:
light-dark-banner

**Review by hyj1204 — APPROVED** [review]
Nice!

**Pull request merged**